lychee.toml

```diff
@@ -60,7 +60,7 @@ insecure = false
 # Only test links with the given schemes (e.g. https).
 # Omit to check links with any other scheme.
 # At the moment, we support http, https, file, and mailto.
-scheme = ["https"]
+scheme = ["https", "file"]
```

Comment:
@brucellino this was skipping internal links so I believe we should have this included